### ISSUE TITLE
Exception running against 'https://github.com/thoughtbot/clearance' in web interface

### ISSUE THREAD

**Issue by nickiemc** [corgibytes/freshli#81]
When running against [https://github.com/thoughtbot/clearance](https://github.com/thoughtbot/clearance) in the web interface, the following exception is thrown:

```
System.NullReferenceException
Object reference not set to an instance of an object.

System.NullReferenceException: Object reference not set to an instance of an object.
   at Freshli.Web.Models.AnalysisRunner.MapResults(IList`1 rawResults, AnalysisRequest request) in /source/Freshli.Web/Models/AnalysisRunner.cs:line 36
   at Freshli.Web.Models.AnalysisRunner.Run(Guid requestGuid) in /source/Freshli.Web/Models/AnalysisRunner.cs:line 22
```

The CLI is able to successfully run against the repository.

**Comment by nickiemc** [comment]
It looks like the issue will be fixed as [part](https://github.com/corgibytes/freshli/pull/61/files#diff-345039a1a158fd9722eba25d65778a63) of PR #61.

**Comment by mscottford** [comment]
@nickiemc Are you able to reproduce this?

**Comment by nickiemc** [comment]
As I suspected, the issues was fixed in PR #61.
